public/packs/manifest.json

```diff
@@ -1,13 +1,13 @@
 {
-  "application.js": "/packs/js/application-67b3a9f1859df37eb11a.js",
-  "application.js.map": "/packs/js/application-67b3a9f1859df37eb11a.js.map",
+  "application.js": "/packs/js/application-a2b635ac00b0a3ad7842.js",
```

Comment:
This file is still going to be removed from this pr right?

Reply:
I thought I had. Can I just delete it from this PR directly using the drop down under the 3 buttons on the top right there?